docker/Dockerfile

```diff
@@ -2,7 +2,7 @@
 # This Dockerfile builds cstor istgt container running istgt from istgt base image
 #
 
-FROM openebs/cstor-ubuntu:xenial-20190515
+FROM openebs/cstor-ubuntu:bionic-20200219
```

Comment:
Has this been tested with the latest ubuntu tag - like `openebs/cstor-ubuntu:xenial-20200212`